src/private/widgets/TabWidgetWidget_p.h

```diff
@@ -68,6 +68,7 @@ class DOCKS_EXPORT TabWidgetWidget
 private:
     void updateMargins();
     void setupTabBarButtons();
+    void contextMenuRequested(QPoint pos);
```

Comment:
make it protected and virtual, so users can create other context menus

Reply:
Done